### ISSUE TITLE
[FR] 

### ISSUE THREAD

**Issue by Gamewalker** [StyraHem/ShellyForHASS#733]
## Is your feature request related to a problem? Please describe.
HA introduced the ability to use shelly v2 devices as bluetooth proxy, but the SFH Addon hides the config option.
https://www.home-assistant.io/blog/2022/12/07/release-202212/#bluetooth-proxy-using-shelly-devices

## Describe the solution you'd like
Add the ability to configure to use shelly devices as BT proxy

## Describe alternatives you've considered
I deactivted the addon and its working fine

**Comment by TheFes** [comment]
You should use the core integration for this, nog ShellyForHass

**Comment by Gamewalker** [comment]
I could be missing somrthing, but the moment I use ShellyForHASS, the core configuration option is gone.

**Comment by ironboy1** [comment]
I agree with @Gamewalker and I would also like to use the BT Proxy function without completely moving back to the core integration.

**Comment by kgroshert** [comment]
I'm also not sure what the options are. I have about 30 shelly devices configured with ShellyForHass and I would have to move them all to the core integration (not sure if everything works there). Having the BT proxy feature would be great. But mixing the core integration for the V2 devices and ShellyForHass for the rest doesn't seem to be possible, or is it?

**Comment by convicte** [comment]
I am usually really annoyed with people writing something down the line of 'same here''; though just in case the devs would like to gauge interest, I was about to open a bug report stating the functionality is missing from the repository.

Thus, I'd like to add myself to the list of interested users.
If there is some workaround for it, I'd be happy to try it out as well, though as mentioned before, the core integration is gone from the panel when this one is put in place.

Thank you in advance!!

**Comment by convicte** [comment]
@TheFes, is there any chance you'd be able to look into this or offer some workarounds as you see them.

Thank you in advance!!

**Comment by TheThunder84** [comment]
I would also love to see this feature, the adjustable polling rate of this integration is great. But the integration would be perfrect with the bluetooth gateway.
